config/locales/sv.yml

```diff
@@ -51,6 +60,14 @@ sv:
               blank: 'Adress krävs, vänligen ange'
             social_media_handle:
               blank: 'Sociala mediehandtag krävs, vänligen ange'
+        interaction:
+          attributes:
+            date:
+              blank: 'Datum krävs, vänligen ange'
```

Comment:
Maybe I can give these to my mum to proofread lol, and maybe @santiagosan93 we can give the Spanish stuff to Lisa 🇪🇸

Reply:
probably not the worst idea no